App/Main/AppDelegate.swift

```diff
@@ -121,13 +124,13 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
         return true
     }
     
-    func applicationWillResignActive(_ application: UIApplication) {
+    func applicationDidEnterBackground(_ application: UIApplication) {
```

Comment:
What's the reasoning here and with DidBecomeActive/WillEnterForeground below? (Not saying it's wrong!)